EMML/1001-2000/EMML1844.xml

```diff
@@ -31,41 +31,252 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
          <sourceDesc>
             <msDesc xml:id="ms">
                <msIdentifier>
-                  <repository ref="INS0004HMML"/>
-                  <collection>Monastery of Ḥayq Esṭifānos, Ambāssal</collection>
-                  <idno>MS EMML no. 1844</idno>
+                  <repository ref="INS0327DHE"/>
+                  <collection>Ethiopian Manuscript Microfilm Library</collection>
+                  <idno>MS EMML no. 1840</idno>
```

Comment:
It requires explanation, why the indo is EMML 1840, while the title of the record is EMML 1844.